frontend/src/app/store/reducers/hydration.reducer.ts

```diff
@@ -0,0 +1,23 @@
+import { ActionReducer, INIT, UPDATE } from '@ngrx/store';
+
+import { AuthState } from '@store/states/auth-state';
+
+export const hydrationMetaReducer = (
```

Comment:
Because of such strange name I have found that you took implementation from https://dev.to/angular/how-to-keep-ngrx-state-on-refresh-2f2o.
Why didn't you take whole implementation? Would be there any benefits to use complete suggested solution?

Reply:
Fixed